lisa/features/hibernation.py

```diff
@@ -18,3 +34,17 @@ def enabled(cls) -> bool:
     @classmethod
     def can_disable(cls) -> bool:
         return True
+
+    @classmethod
+    def _install_by_platform(cls, *args: Any, **kwargs: Any) -> None:
+        raise NotImplementedError()
+
+    @classmethod
+    def _get_resource(cls, resources: Any, type_name: str) -> Any:
```

Comment:
It's used in Azure implementation, so it shouldn't be a method in base Hibernation class. From the name and content, it's hard to understand what kind of resource it returns.